python/cudf/cudf/_lib/table.pyx

```diff
@@ -0,0 +1,46 @@
+from cudf._lib.cudf cimport *
+
+
+cdef class Table:
+    """
+    Python wrapper around cudf::table
+    """
+    def __cinit__(self, columns=None):
```

Comment:
Do we want to handle more than just a list-like of Python columns here? I.E. handle a vector of gdf_column, or handle a cudf_table input?

Reply:
`__cinit__()` can only accept Python arguments so constructors like this can be implemented as `classmethods`, e.g., `Table.from_ptr`